SimPEG/Maps.py

```diff
@@ -519,6 +520,56 @@ def inverse(self, m):
         return np.exp(Utils.mkvc(m))
 
 
+class ChiMap(IdentityMap):
+    """Chi Map
+
+    Convert Magnetic Susceptibility to Magnetic Permeability.
+
+    .. math::
+
+        \mu(m) = \mu_0 (1 + \chi(m))
+
+    """
+
+    def __init__(self, mesh=None, nP=None, **kwargs):
+        super(ChiMap, self).__init__(mesh=mesh, nP=nP, **kwargs)
+
+    def _transform(self, m):
+        return mu_0 * (1 + m)
+
+    def deriv(self, m, v=None):
+        if v is not None:
+            return mu_0 * v
+        return mu_0 * sp.eye(self.nP)
+
+    def inverse(self, m):
+        return m / mu_0 - 1
+
+
+class Mu_relative(IdentityMap):
```

Comment:
cool map. We might want to change `Maps.py` into a folder?

`Maps.EM.Mu_relative` etc.